app/src/main/res/layout/item_library.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<androidx.cardview.widget.CardView
+<androidx.cardview.widget.CardView xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
+  app:cardBackgroundColor="@color/card_view_background"
```

Comment:
you can specify this in the theme, see themes.xml. To understand the complexity of themes here is a [40min video](https://www.youtube.com/watch?v=Owkf8DhAOSo) I link all the time since it was published

Reply:
On it.

Reply:
> you can specify this in the theme, see themes.xml. To understand the complexity of themes here is a [40min video](https://www.youtube.com/watch?v=Owkf8DhAOSo) I link all the time since it was published

The video is really good and helped me understand styles, themes and their differneces.